packages/eslint-config/index.js

```diff
@@ -34,6 +35,13 @@ module.exports = {
         ],
         "import/no-unresolved": [
             "warn"
-        ]
+        ],
+        "no-secrets/no-secrets": [
+            "error",
+            { "additionalRegexes": { 
+                "Basic Auth": "Authorization: Basic [A-Za-z0-9+/=]*",
+                "Common Pattern": "^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*-]).{8,}$"
```

Comment:
the special char may also include ~ | and etc. We may check more.

Reply:
sure